eg/gps-hardware-serial.js

```diff
@@ -0,0 +1,18 @@
+var five = require("../lib/johnny-five.js");
+var board = new five.Board();
+
+board.on("ready", function() {
+
+  /*
+   * Explicitly setting HW_SERIAL1 for the port
+   */
+  var gps = new five.GPS({
+    port: "HW_SERIAL1"
```

Comment:
I'll remove this from program.json. That should have been ```this.io.SERIAL_PORT_IDs.HW_SERIAL1```... I'm starting to rush things and am making mistakes :-(